apps/server/swagger/submission-api.yml

```diff
@@ -56,20 +56,35 @@
       503:
         $ref: '#/components/responses/ServiceUnavailableError'
 
-/submission/{submissionId}/{entityName}:
+/submission/{submissionId}/{actionType}:
   delete:
     summary: Clear Active Submission by entity name
     tags:
-      - submission
+      - Submission
     parameters:
       - name: submissionId
         in: path
         type: string
         required: true
-      - name: entityName
+        description: The ID of the Submission
+      - name: actionType
```

Comment:
if this is listed here, does that make it an additional query param? e.g. `...?actionType=blah`
... wouldn't that be redundant to the endpoint's definition in line 59?

Reply:
It is listed as `in: path`

Reply:
to confirm, does that mean in that third segment of the path, I could provide either of those value types, and the app will somehow know how to identify it?

Reply:
This DELETE endpoint provides the ability to "clean" the Active Submission by choosing the specific item to remove from the Submission. Submission contains a group of staged changes separated on:
- New Uploads(`inserts`),
- Edit Submitted Data(`updates`) and 
- Delete Submitted Data(`deletes`)

Example URL: /submission/102/inserts?entityName=sample&index=0

Where **actionType** is path param to specify the type of record to remove from the Submission. Must be one of: `inserts`, `updates`, or `deletes` (case insensitive)

Thanks for pointing this out. Will fix the description of actionType that is not correct. **